### DISCUSSION TITLE
Exclude certain files from babel cache

### DISCUSSION THREAD

**Discussion by scottdickerson** [babel/babel#15270] [Q&A]
We’re using the babel cache settings to improve our webpack performance considerably. However we still have some Angular files in our codebase and references from HTML files to angular controllers are not being reliably updated in the babel cache. We have to manually clean it to fix the issue. Is there a way to exclude certain file patterns from the cache or force those files to always update in the cache?

**Comment by liuxingbaoyu** [comment]
https://github.com/babel/babel-loader/pull/971
Does this help? (you can take a look at the newly added test)
